src/app/admin/get-started/get-started.component.ts

```diff
@@ -54,19 +59,37 @@ export class GetStartedComponent implements OnInit {
     this.checkEntity();
   }
 
-  checkthebox(){
+  changeEvent3() {
+    this.checkbox3 = !this.checkbox3;
+    if (this.checkbox3) {
+      this.colorBorder3 = "border-color-sec";
+    } else {
+      this.colorBorder3 = "border2";
+    }
+    this.checkEntity();
+  }
+
+  checkthebox() {
     console.log("checked")
   }
 
   checkEntity() {
-    if (this.checkbox1 == true && this.checkbox2 == false) {
-      this.chooseEntity = "atstandclaim";
-    } else if (this.checkbox1 == false && this.checkbox2 == true) {
-      this.chooseEntity = "issuance";
-    } else if (this.checkbox1 == false && this.checkbox2 == false) {
-      this.chooseEntity = "new";
-    } else if (this.checkbox1 == true && this.checkbox2 == true) {
-      this.chooseEntity = "newcombination";
+    if (this.checkbox1 == true && this.checkbox2 == false && this.checkbox3 == false) {
+      this.chooseEntity = "attestmodule";
+    } else if (this.checkbox1 == false && this.checkbox2 == true && this.checkbox3 == false) {
+      this.chooseEntity = "vcmodule";
+    } else if (this.checkbox1 == false && this.checkbox2 == false && this.checkbox3 == true) {
+      this.chooseEntity = "ownershipmodule";
+    } else if (this.checkbox1 == true && this.checkbox2 == true && this.checkbox3 == false) {
```

Comment:
@suvarnakale can we reduce this conditions? Check once we can achive this usign another approch ? or not